### PR TITLE
Bring run_gui_tests.sh back

### PR DESCRIPTION
**Pull request by atodorov**

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -202,6 +202,21 @@ ci: rc-release
...
+test-gui:
+	@rm -f tests/test-suite.log
+	@rm -rf tests/autogui-results-*/
+	@sudo mkdir -p $(ROOT_SITE_PACKAGES)
+	@sudo cp $(abs_builddir)/tests/usercustomize.py $(ROOT_SITE_PACKAGES)
+# We require SELinux in Permissive mode, see rhbz#1276376
+	@sudo setenforce 0
+	@sudo COVERAGE_PROCESS_START=$(abs_builddir)/.coveragerc \
+		$(abs_builddir)/tests/run_gui_tests.sh >tests/test-suite.log 2>&1
+	@sudo setenforce 1
```
Comment (clumens) [inline Makefile.am]: I don't think you should worry about calling setenforce here and on line 211.  First off, we're already testing for it with that decorator on DogtailTestCase.  Second, if you just unconditionally "setenforce 1" here, you'll be enabling it on systems that might have been set to permissive in the first place.

Comment (atodorov) [inline Makefile.am]: OK, I agree I might turn selinux on if it was turned off. But Dogtail tests will not run of I don't call setenforce 0. So I will simply restore whatever SELinux state was there previously.

**Comment by atodorov** [comment]
Hi Chris,
objections on this one ? If not I will rebase so you can merge it without conflicts.

**Comment by clumens** [comment]
I still don't think it's the test infrastructure's place to do anything with selinux.  I'd prefer that to just be a prerequisite to running tests in the first place - if you're setting up a machine to run these tests, you just need selinux in permissive mode.  If anything, I'd prefer it if we could use the same decorators to fail out/skip if selinux isn't set the way we want, just like we do for other things.

**Comment by atodorov** [comment]
Hi Chris,
I've removed the bits where I modify the SELinux config. Note however that the system which will be executing these tests (aka the Jenkins box) needs to have SELinux configured properly, otherwise all the gui tests will be skipped which is pretty much the same as not executing them at all.

**Comment by clumens** [comment]
Yeah that's not a problem - I've got to have selinux off for various reasons anyway.

**Comment by clumens** [comment]
Merged, thanks.  I guess I need to add a jenkins job back for this, huh?  Also is there anything I need to verify is installed on the jenkins builders?

**Comment by atodorov** [comment]
> I guess I need to add a jenkins job back for this, huh? 

well yes :)

> Also is there anything I need to verify is installed on the jenkins builders?

I don't think so. Everything needed should already be in install-test-requires so you should be all set.

**Comment by clumens** [comment]
Alright sounds good.  I've re-enabled the jenkins job and made sure packages are installed and sudo is configured.  It should kick off overnight (for me).  I'm sure it'll fail, and tomorrow I'll figure out what I didn't set up right.

**Pull request closed without merging**